<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAbout</class>
 <widget class="QWidget" name="DlgAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于CKFontEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lab_icon">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="pixmap">
          <pixmap resource="../res.qrc">:/res/icon.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lab_title">
         <property name="text">
          <string notr="true">CFontEditor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_desc">
         <property name="text">
          <string>ChenKe404的字体编辑器</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lab_version">
         <property name="text">
          <string notr="true">版本/Qt版本</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_build_date">
         <property name="text">
          <string notr="true">构建时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_licence">
         <property name="text">
          <string notr="true">证书类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_repo">
         <property name="text">
          <string>仓库:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_repos">
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_ok</sender>
   <signal>clicked()</signal>
   <receiver>DlgAbout</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
